### PR TITLE
EntityStorage empties contents on destroy.

### LINKED ISSUE #1070: Lockers don't drop contents when destroyed
For example - shoot a locker with a gun, and its contents will be destroyed with it.

### PR DESCRIPTION
**Pull request by FL-OZ**
fixes #1070

### CODE REVIEW

File: Content.Server/GameObjects/Components/Damage/DestructibleComponent.cs [hunk 1]
```diff
@@ -73,6 +73,10 @@ void IOnDamageBehavior.OnDamageThresholdPassed(object obj, DamageThresholdPassed
             {
                 destroyed = true;
                 var pos = Owner.Transform.GridPosition;
+                if(Owner.TryGetComponent<EntityStorageComponent>(out var entityStorage))
```
Comment (ComicIronic) [inline Content.Server/GameObjects/Components/Damage/DestructibleComponent.cs:76]: I think you should just make `EntityStorageComponent` handle `IDestroyAct`.

Comment (FL-OZ) [reply Content.Server/GameObjects/Components/Damage/DestructibleComponent.cs:76]: WIll do.

**Comment by ComicIronic** [comment]
This kind of mini-PR really demonstrates how good the codebase is. Lots of feature in 6 lines.

**Comment by FL-OZ** [comment]
> This kind of mini-PR really demonstrates how good the codebase is. Lots of feature in 6 lines.

I agree

**Pull request merged**